pkg/investigations/CannotRetrieveUpdatesSRE/metadata.yaml

```diff
@@ -0,0 +1,20 @@
+name: cannotretrieveupdatessre
+rbac:
+  roles: []
+  clusterRoleRules:
+    - verbs:
+        - "get"
+        - "list"
+      apiGroups:
+        - "config.openshift.io"
+      resources:
+        - clusteroperators
+    - apiGroups:
+        - ""
+      resources:
+        - pods
+        - namespaces
+      verbs:
+        - get
+        - list
```

Comment:
Does this actually need a ClusterRole and `get/list` on `pods/namespaces` for the _entire_ cluster? This should be scoped as tightly as we can right?

Reply:
I had added it as per ref Alice talked about last week but ill perform a trial and error and if during the testing time, i will need it then ill add the get/list on pods/namespaces then